go.mod

```diff
@@ -9,6 +9,7 @@ require (
 
 require (
 	github.com/davecgh/go-spew v1.1.1 // indirect
+	github.com/jstemmer/go-junit-report v1.0.0 // indirect
```

Comment:
We can do `go install github.com/jstemmer/go-junit-report/v2@latest` in the workflow, rather than adding it as a dependency here, which will impact the customer.

Reply:
it got auto added when i tried to install the go-junit-report pkg 
will resolve this

Reply:
Thank you!